downstairs/src/dynamometer.rs

```diff
@@ -0,0 +1,204 @@
+// Copyright 2023 Oxide Computer Company
+use super::*;
+
+pub enum DynoFlushConfig {
+    FlushPerIops(usize),
+    FlushPerBlocks(usize),
+    FlushPerMs(usize),
```

Comment:
To make this more strongly typed, we could do `FlushesPerDuration(std::time::Duration)`

Reply:
done in e9f076d3de969cefdea895ca04944cea2f62f864